.vib/joomla/cypress/cypress/integration/joomla_spec.js

```diff
@@ -0,0 +1,72 @@
+/// <reference types="cypress" />
+import { random } from './utils';
+
+it('allows user to log in and log out', () => {
+  cy.login();
+  cy.contains('Username and password do not match').should('not.exist');
+  cy.contains('User Menu').click();
+  cy.contains('Log out').click();
+  cy.get("[name='username']");
```

Comment:
I would rather search for the whole login form here for additional clarity when reading the test.